kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-swagger/src/main/java/org/kie/server/remote/rest/swagger/HistorySwaggerDecoration.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.remote.rest.swagger;
+
+import static java.util.Arrays.asList;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import javax.ws.rs.GET;
+
+import org.kie.server.remote.rest.common.marker.KieServerEndpoint;
+import org.kie.server.remote.rest.common.marker.KieServerEndpoint.EndpointType;
+
+import io.swagger.jaxrs.ext.AbstractSwaggerExtension;
+import io.swagger.jaxrs.ext.SwaggerExtension;
+import io.swagger.models.Operation;
+
+
+public class HistorySwaggerDecoration extends AbstractSwaggerExtension {
+
+    public static final String SWAGGER_EXTENSION_READ_ONLY = "x-readonly"; 
```

Comment:
Maybe `x-history` makes more sense here? not sure, wdyt?

```suggestion
    public static final String SWAGGER_EXTENSION_READ_ONLY = "x-history"; 
```

Reply:
not really. At first I thought history was the good one, but all operations are read only for database. I think makes more sense.